flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/PartitionLoader.java

```diff
@@ -154,8 +166,82 @@ private void moveFiles(List<Path> srcDirs, Path destDir) throws Exception {
         }
     }
 
+    /**
+     * Reuse of PartitionCommitPolicy mechanisms. The default in Batch mode is metastore and
```

Comment:
The comment is not correct. For filesystem, the default is nothing.
I think we can just explain what this method do.

Reply:
this will be remove.